cots/example_test.go

```diff
@@ -1,3 +1,5 @@
+// Copyright 2024 Contributors to the Veraison project.
```

Comment:
Are these old files with missing Copyright Headers ?

If yes, then we should introduce the 2023-2024, as these files were introduced by Sabreen in 2023